osu.Game/Rulesets/Edit/Checks/CheckPreviewTime.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Edit.Checks.Components;
+
+namespace osu.Game.Rulesets.Edit.Checks
+{
+    public class CheckPreviewTime : ICheck
+    {
+        public CheckMetadata Metadata => new CheckMetadata(CheckCategory.Timing, "Check Preview Time Consistency");
```

Comment:
Doesn't stylistically fit the rest of the checks wording-wise (and also wrt casing). Suggest [following BeatmapVerifier](https://github.com/Naxesss/MapsetChecks/blob/8efbf76d17aa495732b2de0b75e33132169cecd4/Checks/AllModes/Timing/CheckPreview.cs#L15):

```suggestion
        public CheckMetadata Metadata => new CheckMetadata(CheckCategory.Timing, "Inconsistent or unset preview time");
```